src/tools/concurrency/ActorExecutionTrace.java

```diff
@@ -210,6 +211,7 @@ public static final void forceSwapBuffers() {
 
   protected enum Events {
     ActorCreation(TraceData.ACTOR_CREATION,         19),
+    ActorCreationWithOrigin(TraceData.ACTOR_CREATION_ORIGIN,         27),
```

Comment:
Just for change documentation: I think, I'll change the encoding here, and write after the normal actor creation event, directly following, a creation origin. That avoids having everything three times.